plugins/org.locationtech.udig.ui/src/org/locationtech/udig/ui/graphics/SWTGraphics.java

```diff
@@ -353,12 +353,14 @@ public void setTransform( AffineTransform transform ) {
         transform.getMatrix(matrix);
         if( swtTransform==null ){
             swtTransform=new Transform(display, 
-                    (float)matrix[0], (float)matrix[1], (float)matrix[2], 
-                    (float)matrix[3], (float)matrix[4], (float)matrix[5] );
+                    (float)matrix[0], (float)matrix[2],
+                    (float)matrix[1], (float)matrix[3],
+                    (float)matrix[4], (float)matrix[5] );
         }else{
             swtTransform.setElements(
-                    (float)matrix[0], (float)matrix[1], (float)matrix[2], 
-                    (float)matrix[3], (float)matrix[4], (float)matrix[5] );
+                    (float)matrix[0], (float)matrix[2],
+                    (float)matrix[1], (float)matrix[3],
+                    (float)matrix[4], (float)matrix[5] );
```

Comment:
same comment here

Reply:
Transform.setElements(...) expects the arguments in the same order as the constructor. (see above)